### DISCUSSION TITLE
Images not showing

### DISCUSSION THREAD

**Discussion by Nergl02** [requarks/wiki#7108] [Error / Bug Report]
Hello, my images are not showing, it was fine, but does anyone know why?
![image](https://github.com/requarks/wiki/assets/116952000/ec0620d9-ce3d-4dd1-a50e-3f08de1f6308)

Images are still uploaded!

**Comment by P4mes4n** [comment]
It looks like you uploaded your images at the root-level. You need the read:assets permission on the path where you stored the images.

So you need to either move the images to the same path as your page (assuming you have the correct permissions set for that path) or give the read:asset permission for the root path.

**Comment by jcdufourd** [comment]
I had the same problem of images not appearing, and solved it by changing the "Path" of the "Local File System" in the "Storage" section of the admin area from **_relative_** to **_absolute_** (as required in the comment below the field)
